delfin/drivers/ibm/ds8k/ds8k.py

```diff
@@ -259,3 +262,131 @@ def clear_alert(self, context, alert):
     @staticmethod
     def get_access_url():
         return 'https://{ip}:{port}'
+
+    def get_serial_number(self):
+        serial_number = None
+        system_info = \
+            self.rest_handler.get_rest_info('/api/v1/systems')
+        if system_info:
+            system_data = system_info.get('data', {}).get(
+                'systems', [])
+            if system_data:
```

Comment:
if 'system_data' is of type list, do not add 'if system_data'

Reply:
done